x-pack/plugins/serverless_search/public/application/components/overview_panels/select_client.tsx

```diff
@@ -0,0 +1,181 @@
+import { FormattedMessage } from '@kbn/i18n-react';
+import React from 'react';
+import { PLUGIN_ID } from '../../../../common';
+
+import { useKibanaServices } from '../../hooks/use_kibana';
+import { LanguageDefinition, Languages } from '../languages/types';
+import { OverviewPanel } from './overview_panel';
+import { docLinks } from '../../../../common/doc_links';
+interface SelectClientProps {
+  setSelectedLanguage: (language: LanguageDefinition) => void;
+  languages: LanguageDefinition[];
+  selectedLanguage: LanguageDefinition;
+}
+export const SelectClientPanel: React.FC<SelectClientProps> = ({
+  setSelectedLanguage,
+  languages,
+  selectedLanguage,
+}) => {
+  const { http } = useKibanaServices();
+  const { euiTheme } = useEuiTheme();
+  const panelItems = languages.map((language) => (
+    <EuiFlexGroup direction="column">
+      <EuiFlexItem grow={false}>
+        <EuiPanel
+          hasBorder
+          borderRadius="m"
+          onClick={() => setSelectedLanguage(language)}
+          grow={false}
+          paddingSize="m"
+          css={
+            selectedLanguage === language
+              ? css`
+                  border: 1px solid ${euiTheme.colors.primary};
+                `
+              : css`
+                  border: 1px solid ${euiTheme.colors.lightShade};
+                `
+          }
+          color={selectedLanguage === language ? 'primary' : 'plain'}
+        >
+          <EuiFlexGroup direction="column" gutterSize="l" justifyContent="center">
+            <EuiFlexItem>
+              <EuiImage
+                alt=""
+                src={http.basePath.prepend(`/plugins/${PLUGIN_ID}/assets/${language.iconType}`)}
+                height="100px"
+                width="128px"
+              />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiText
+                textAlign="center"
+                color={selectedLanguage === language ? 'default' : 'subdued'}
+              >
+                <h5>
+                  <FormattedMessage
+                    id="xpack.serverlessSearch.selectClient.language.name"
+                    defaultMessage="{languageName}"
+                    values={{
+                      languageName: language.name === Languages.CURL ? 'cURL' : `${language.name}`,
+                    }}
+                  />
+                </h5>
+              </EuiText>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiPanel>
+      </EuiFlexItem>
+    </EuiFlexGroup>
+  ));
+
+  return (
+    <OverviewPanel
+      description={
+        <FormattedMessage
+          id="xpack.serverlessSearch.selectClient.description"
+          defaultMessage="Elastic builds and maintains clients in several popular languages and our community has contributed many more. Select your favorite language client or dive into the {console} to get started."
+          values={{
+            console: (
+              <EuiLink href="../app/dev_tools#/console">
```

Comment:
we should not use `../` in a `href` you can use `http.basePath.prepend()`, [see an example here](https://github.com/elastic/kibana/blob/6b543e7ab2fd24ab9c4927ad1423f3d423b2876f/x-pack/plugins/serverless_search/public/application/components/api_key/api_key.tsx#L120), to build an `href` to another app.